test/smokes/clang_tidy/expectations.rb

```diff
@@ -0,0 +1,224 @@
+s = Runners::Testing::Smoke
+
+s.add_test(
+  "success",
+  type: "success",
+  issues: [
+    {
+      path: "foo/テスト.cpp",
+      location: { start_line: 6, start_column: 5 },
+      id: "clang-analyzer-core.CallAndMessage",
+      message: "2nd function call argument is an uninitialized value",
+      object: {
+        severity: "warning"
+      },
+      git_blame_info: nil,
+      links: []
+    },
+    {
+      path: "test.c",
+      location: { start_line: 5, start_column: 3 },
+      id: "clang-analyzer-core.uninitialized.Assign",
+      message: "Assigned value is garbage or undefined",
+      object: {
+        severity: "warning"
+      },
+      git_blame_info: nil,
+      links: []
+    },
+    {
+      path: "test.c",
+      location: { start_line: 5, start_column: 15 },
+      id: "clang-analyzer-deadcode.DeadStores",
+      message: "Value stored to 't' during its initialization is never read",
+      object: {
+        severity: "warning"
+      },
+      git_blame_info: nil,
+      links: []
+    }
+  ],
+  analyzer: { name: "Clang-Tidy", version: "10.0.1" }
```

Comment:
[suggestion] How about making a variable as follows to reduce diffs when bumping the version?

https://github.com/sider/runners/blob/22c3d8e3ac3972e7aa12d36780ca07970028c847/test/smokes/eslint/expectations.rb#L3

```suggestion
  analyzer: { name: "Clang-Tidy", version: default_version }
```

Reply:
Fixed. commit 334b6dbbb8f51c0c4d9500e853e88c617cbc7649